src/site/markdown/Examples.md

```diff
@@ -30,6 +30,11 @@ are marked with a trailing star (*).
 ...
 ```
 
+At your preference, japicmp can generate a [Markdown](https://www.markdownguide.org/) report.
```

Comment:
Can you move this section below the HTML section, i.e. between HTML and XML? :smirk:

Reply:
Sure https://github.com/siom79/japicmp/pull/405/commits/98fe35bcd0d6c509f8dccf5231631f91d784844e